### PR TITLE
Azure > IAC -> Rules + Test Case Fix (details in the description)

### PR DESCRIPTION
**Pull request by rezoan**
Fixes
- applicationgateways.rego
- blobservices.rego
- databaseaccounts.rego
- dbadministrators.rego
- dbauditingsettings.rego
- dbdataencryption.rego
- dbfirewallrules.rego
- dbsecurityalertpolicies.rego
- dbvulnerabilityassesments.rego
- master-complience-test.json (for test cases)

### CODE REVIEW

File: azure/iac/blobservices.rego [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+storage_blob_soft_delete = false {
+    azure_issue["storage_blob_soft_delete"]
+}
+
+storage_blob_soft_delete_err = "Soft delete on blob service is not enabled" {
+    azure_issue["storage_blob_soft_delete"]
+}
+
+storage_blob_soft_delete_metadata := {
+    "Policy Code": "",
```
Comment (farchide) [inline azure/iac/blobservices.rego:37]: Policy Code should be entered
and master compliance json should be updated with the new policy

File: azure/iac/blobservices.rego [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+storage_blob_container_soft_delete = false {
+    azure_issue["storage_blob_container_soft_delete"]
+}
+
+storage_blob_container_soft_delete_err = "Soft delete on blob service container is not enabled" {
+    azure_issue["storage_blob_container_soft_delete"]
+}
+
+storage_blob_container_soft_delete_metadata := {
+    "Policy Code": "",
```
Comment (farchide) [inline azure/iac/blobservices.rego:81]: Policy Code should be entered
and master compliance json should be updated with the new policy

File: azure/iac/dbdataencryption.rego [hunk 3]
```diff
@@ -34,11 +34,11 @@ db_encrypt = false {
     azure_attribute_absence["db_encrypt"]
 }
 
-db_encrypt_err = "SQL databases has encryption disabled" {
+db_encrypt_err = "Azure SQL databases currently dont have transparent data encryption enabled" {
     azure_issue["db_encrypt"]
 }
 
-db_encrypt_miss_err = "DB encryption attribute status missing in the resource" {
+db_encrypt_miss_err = "zure SQL databases transparent encryption attribute 'status' is missing from the resource" {
```
Comment (farchide) [inline azure/iac/dbdataencryption.rego:41]: typo "Azure"

**Review by farchide — CHANGES_REQUESTED** [review]

File: azure/iac/blobservices.rego (hunk 1 above)
Comment (rezoan) [reply azure/iac/blobservices.rego:37]: done.

File: azure/iac/blobservices.rego (hunk 2 above)
Comment (rezoan) [reply azure/iac/blobservices.rego:81]: done.

File: azure/iac/dbdataencryption.rego (hunk 3 above)
Comment (rezoan) [reply azure/iac/dbdataencryption.rego:41]: done.

**Review by farchide — APPROVED** [review]

**Pull request merged**